test/scripts/utils/accounts.ts

```diff
@@ -63,15 +63,16 @@ export const createAccount = async () => {
 	};
 };
 
-const passphrases = [
-	'attract squeeze option inflict dynamic end evoke love proof among random blanket table pumpkin general impose access toast undo extend fun employ agree dash',
-];
+const passphrase =
+	'attract squeeze option inflict dynamic end evoke love proof among random blanket table pumpkin general impose access toast undo extend fun employ agree dash';
 
-const accountKeyPath = "m/44'/134'/0'";
+export const getAccountKeyPath = () => {
+	const accountKeyPathOffset = Math.floor(Math.random() * 102);
```

Comment:
I'd recommend using `103` because `0 <= Math.random() < 1`. 
```suggestion
	const accountKeyPathOffset = Math.floor(Math.random() * 103);
```